clubs_guides/urls.py

```diff
@@ -1,3 +1,18 @@
 from django.conf.urls import url
 
-# Create you url mappings here.
+from clubs_guides.views import (
+    ClubsGuidesListView,
+    ClubsGuideView,
+    CategoryListView,
+)
+
+
+urlpatterns = [
+    url("^$", ClubsGuidesListView.as_view(), name="clubsguide-list"),
+    url(r"^(?P<pk>[0-9]+)", ClubsGuideView.as_view(), name="clubsguide"),
+    url(
```

Comment:
I can split this over 2 lines (1 line is too long). I just thought it looked better this way.